README.MD

```diff
@@ -88,6 +88,37 @@ Presidio leverages:
 * [GRPC](https://grpc.io)
 * [re2](https://github.com/google/re2)
 
+## Give it a try - Run locally with KIND
+Presidio is built for Kubernetes, you can give it a try using [KIND (Kubernetes IN Docker)](https://github.com/kubernetes-sigs/kind).
+
+1. Install [Docker](https://docs.docker.com/install/).
+    * **Optional (Linux)** - the following command will install all prerequisites (Docker, Helm, make, kubetl).
+
+        ```sh
+        cd deployment/
+        ./prerequisites.sh
+        ```
+
+        depending on your environment, sudo might be needed
+
+2. Clone Presidio.
+3. Run
```

Comment:
Could you add a sentence on what this script does?

Reply:
done